src/components/Steps.tsx

```diff
@@ -24,11 +25,24 @@ const Steps = () => {
 
     const handleConnectWallet = () => popupsActions.setIsConnectorsWalletOpen(true)
 
+    const sendConnectWalletEvent = async (walletAddress: string): Promise<void> => {
```

Comment:
Is this meant to be duplicated with the useFuulSdk hook?

Reply:
It shouldn't be duplicated we should use the method in the hook @jahabeebs are you handling it or should I?

Reply:
> It shouldn't be duplicated we should use the method in the hook @jahabeebs are you handling it or should I?

I fixed it